### ISSUE TITLE
Shall we merge?

### ISSUE THREAD

**Issue by agstephens** [cedadev/objstore_lister#2]
@nmassey001 shall we merge this and [cedadev/jos](https://github.com/cedadev/jos/)?

I have been testing `minio` and the combination of the command-line `mc` and the python `minio` library gives me everything except a couple of extras that I can wrap in a few lines of python and present as functions or `cli` utilities. 

I suspect that `objstore_lister` is going to be one of such functions. If we merge then we can piggyback on each others extensions. I have unit tests :-)

**Comment by nmassey001** [comment]
@agstephens Yes, it looks sensible to merge [objstore_lister](https://github.com/cedadev/objstore_lister/) and [cedadev/jos](https://github.com/cedadev/jos/).  objstore_lister is just a quick script that Sam started, for a user query, and I converted to using minio and got it to work fully.

I'm all for minimising everything! :)  However, I think the trap here is wrapping everything that mc does in a new tool.
After a bit of a play with `mc` I realise that I could have got the same output with:

`mc ls ceda-archive-o/spot-36786-2008 -r --json`
